src/utilities/theme/index.ts

```diff
@@ -1,14 +1,10 @@
-export {ThemeContext} from './context';
+export * from './context';
 
-export {useTheme} from './hooks';
+export * from './hooks';
 
-export {Theme, ThemeConfig, CustomPropertiesLike, ColorScheme} from './types';
+export {ThemeConfig} from './types';
```

Comment:
Why not export * here?

Reply:
There's a bunch of other types  in that file that are used within this folder, but not externally. As most of this is going to be extracted into polaris-tokens pretty soon it didn't seem worth the refactoring to split this into a types and private-types file.